depclean-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,50 @@
+//buildscript{
```

Comment:
Can you remove the comments?

Reply:
Ok.

Reply:
you uncommented, you should remove those lines. if the plugin is not present on the system, it is not possible to compile it.

Reply:
Sorry that was my mistake.
I will correct it in my next push.